cloud/cloud_scheduler_test.cc

```diff
@@ -34,7 +40,7 @@ class CloudSchedulerTest : public testing::Test {
         }
       }
       if (running) {
-        usleep(delay);
+        env_->SleepForMicroseconds(delay);
```

Comment:
You could also use `std::this_thread::sleep_for` from C++ standard.

Reply:
I could have used sleep_for, but chose to use the RocksDB functions, which also support fake sleeps if we choose to use one.